### PR TITLE
Feat: Adds the Info snackbar when the About info is copied

### PR DESCRIPTION
**Pull request by panoramix360**
#### Summary

This PR adds the Snackbar information when the user presses the `Copy info` button inside the About page.

#### Ticket Link

[Github Issue](https://github.com/mattermost/mattermost/issues/24131)

#### Checklist

- [X] Has UI changes
- [X] Includes text changes and localization file updates
- [X] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
This PR was tested on: 
- iOS Simulator, iOS 16.4
- Android Emulator, Nexus 5 API 31

#### Screenshots

# iOS
image

# Android
image

#### Release Note

```release-note
Adds a Snackbar Info when the `Copy Info` button is clicked on the About screen
```

### CODE REVIEW

File: app/screens/settings/about/about.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ import {useIntl} from 'react-intl';
 import {Alert, Text, View} from 'react-native';
 import DeviceInfo from 'react-native-device-info';
 
+import {SNACK_BAR_TYPE} from '@app/constants/snack_bar';
+import {showSnackBar} from '@app/utils/snack_bar';
```
Comment (larkox) [inline app/screens/settings/about/about.tsx:11]:
```suggestion
import {SNACK_BAR_TYPE} from '@constants/snack_bar';
import {showSnackBar} from '@utils/snack_bar';
```

File: assets/base/i18n/es.json [hunk 2]
```diff
@@ -585,6 +585,7 @@
     "settings.save": "Guardar",
     "smobile.search.recent_title": "Búsquedas recientes en {teamName}",
     "snack.bar.favorited.channel": "Este canal estaba como favorito",
+    "snack.bar.info.copied": "El info ha sido copiado al portapapeles",
```
Comment (larkox) [inline assets/base/i18n/es.json:588]: Thank you for adding the translations. Nevertheless, they are handled by a different system, so it is better for us if you only add the English string. Same for the brazilian translation.

**Review by larkox — CHANGES_REQUESTED** [review]
Great work! Thank you very much for the contribution. Just a few minor changes.

File: app/screens/settings/about/about.tsx (hunk 1 above)
Comment (panoramix360) [reply app/screens/settings/about/about.tsx]: Fixed! Thank you!

File: assets/base/i18n/es.json (hunk 2 above)
Comment (panoramix360) [reply assets/base/i18n/es.json]: Didn't know about this, thank you for explaining!

**Review by enahum — APPROVED** [review]

**Review by larkox — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]
thank you for the changes 🎉 

rn_image_picker_lib_temp_d8df82d3-78c6-47c5-92ee-d4c2295aef28
rn_image_picker_lib_temp_58119ba2-db59-4900-925e-5c11ae01303c

**Comment by larkox** [comment]
@panoramix360 Thank you for the contribution!

**Pull request merged**